crates/hyperdrive-math/src/lib.rs

```diff
@@ -142,6 +142,21 @@ impl State {
         }
     }
 
+    /// Gets the normalized time remaining
```

Comment:
This seems like a slight variation of the function above it, `time_remaining_scaled`; I think we should just have one of these to avoid confusion.